src/Language/PureScript/CoreImp/Optimizer/MagicDo.hs

```diff
@@ -102,10 +93,10 @@ magicDo effectModule C.EffectDictionaries{..} expander = everywhereTopDown conve
 inlineST :: AST -> AST
 inlineST = everywhere convertBlock
   where
-  -- Look for runST blocks and inline the STRefs there.
-  -- If all STRefs are used in the scope of the same runST, only using { read, write, modify }STRef then
+  -- Look for run blocks and inline the STRefs there.
+  -- If all STRefs are used in the scope of the same run, only using { read, write, modify } then
```

Comment:
Should these docs still say `runST` and `STRef`?

Reply:
The identifiers it's looking for are literally named [`run`](https://pursuit.purescript.org/packages/purescript-st/6.2.0/docs/Control.Monad.ST.Internal#v:run), [`read`](https://pursuit.purescript.org/packages/purescript-st/6.2.0/docs/Control.Monad.ST.Internal#v:read), etc. I figured the reason these comments were the way they were was because the declarations in the Constants.Prelude module were
```hs
runST :: forall a. (IsString a) => a
runST = "run"
```
etc.

I changed the naming convention so I changed the comments to match.

Reply:
Ok. Thanks for clarfying.